lang/main-sv.json

```diff
@@ -1159,7 +1342,7 @@
         "go": "KÖR",
         "goSmall": "BÖRJA",
         "headerSubtitle": "Säkra möten med hög kvalitet",
-        "headerTitle": "Jitsi Meet",
+        "headerTitle": "Jitsi meet",
```

Comment:
Can you drop this line, sorry it just caught my eye. There is no point in having it when it will duplicate the English one (and the correct one should be the caps Meet).